### DISCUSSION TITLE
Is it a typo in releases page: `python -m python`

### DISCUSSION THREAD

**Discussion by waketzheng** [fastapi/fastapi#11955] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
nothing
```

### Description

Description of [0.112.0](https://github.com/fastapi/fastapi/releases/tag/0.112.0) may be have a typo (should be `python -m fastapi`):
<img width="1160" alt="image" src="https://github.com/user-attachments/assets/16eae768-e264-4e48-a4b9-04f2351139a5">

### Operating System

Other

### FastAPI Version

0.112.0

### Pydantic Version

N/A

### Python Version

N/A

**Comment by AIdjis** [comment]
I think it is just a mistake from the author. The correct command is from https://github.com/fastapi/fastapi/pull/11935:
```python -m fastapi```

**Comment by waketzheng** [comment]
@alejsdev Could you edit the release note?

**Comment by tiangolo** [comment, marked as answer]
Yes, it was a typo! Good catch, thanks! :cake:
